eng/pipelines/branch-cleanup.yml

```diff
@@ -11,7 +11,16 @@ parameters:
     - azure-sdk-for-js
     - azure-sdk-for-net
     - azure-sdk-for-python
-
+- name: AzureDocsRepo
```

Comment:
Why add another set of parameters? If the only thing different is a branch pattern we should just add that as a property of the repo so we can use the same logic for everything.

Reply:
Something similar to https://github.com/Azure/azure-sdk-tools/blob/main/eng/pipelines/mirror-repos.yml#L21